### PR TITLE
update arrow config

### PR DESCRIPTION
**Pull request by mathilde-daugy**
## Description

Update configs for exchanges for:

- US-CENT-SWPP->US-MIDW-MISO (overlapping and in the wrong place)
- US-CENT-SWPP->US-MIDW-AECI (overlapping and in the wrong place)
image

- AR->PY (not on the border)
- AR-> CL-SEN (in the sea)

image

### Preview

NA

### CODE REVIEW

**Review by VIKTORVAV99 — COMMENTED** [review]
Could we hold of on the changes to AR exchanges?
They are fixed in #4894

If not could we update this to use the same coordinates and rotations so there are no conflicts?

**Comment by mathilde-daugy** [comment]
> Could we hold of on the changes to AR exchanges? They are fixed in #4894
> 
> If not could we update this to use the same coordinates and rotations so there are no conflicts?

Ok thanks for flagging, we'll be using the config in the other PR then

File: config/exchanges/US-CENT-SWPP_US-SW-EPE.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 lonlat:
-  - -103.81455
+  - -1000.4
```
Comment (tonypls) [inline config/exchanges/US-CENT-SWPP_US-SW-EPE.yaml:2]: I this meant to be 1000?

**Comment by tonypls** [comment]
These two need to be flipped 180 degrees 
image

**Review by tonypls — APPROVED** [review]
LGTM 🦦

**Pull request merged**